java/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.4)
```

Comment:
That's the minimum version of cmake that supports `create_javah`. See https://cmake.org/cmake/help/latest/release/3.4.html?highlight=create_javah.